blockchain/fundamentals/solidity/ether-heists-caused-by-bugs.md

```diff
@@ -11,23 +11,36 @@ type: normal
 
 category: must-know
 
+standards:
+  blockchain.identify-blockchain-applications.4: 10
+
 aspects:
 
   - introduction
 
   - deep
 
+links: 
+  - '[TheDAO hack](https://www.coindesk.com/understanding-dao-hack-journalists/){article}'
+
 ---
 # Ether Heists Cause by Bugs
```

Comment:
is there solidity specific that you can mention in this one?

Reply:
something*

Reply:
there are some examples of overflows that can happen, but not as eye-catching as theDao